### ISSUE TITLE
Setup github actions publication with io.github.avro-kotlin package name

### ISSUE THREAD

**Issue by sksamuel** [avro-kotlin/avro4k#84]
https://issues.sonatype.org/browse/MVNCENTRAL-6314

**Comment by sksamuel** [comment]
@thake if you add your username to that JIRA you will get permissions too.

**Comment by thake** [comment]
@sksamuel I tried to, but I can't view the issue. Can you add my username? It is also thake in sonatype.

**Comment by sksamuel** [comment]
I've updated the ticket with 'thake'

**Comment by thake** [comment]
@sksamuel thanks!

**Comment by geomagilles** [comment]
Hi - is this related to the ability to do a new release? I'm asking bluntly because I need this new release for my own project 😬. Let me know if I can help

**Comment by sksamuel** [comment]
They had me make a new ticket, so waiting on this one now:
https://issues.sonatype.org/browse/OSSRH-62126
Hopefully within 24 hours.

**Comment by sksamuel** [comment]
I've released 1.0.0 on com.github.avro-kotlin.
https://search.maven.org/search?q=com.github.avro-kotlin

**Comment by thake** [comment]
I still don't have access to publish to com.github.avro-kotlin. Tried to create a new ticket in OSSRH but that was not successful. Next try is to comment on the existing ticket OSSRH-62126 and hope that someone sees it.

**Comment by sksamuel** [comment]
Was access setup ok (is that why you closed this?) @thake

**Comment by thake** [comment]
Yes, that's the reason. Sorry, forgot to comment on it.
